### ISSUE TITLE
composer local repository issue

### ISSUE THREAD

**Issue by Ylmzef** [markshust/docker-magento#897]
**Description**
I want to add my local repositories folder to the composer but it gives error.

**Steps To Reproduce**
1.  bin/composer config repo.theme path repos/package-name
2. bin/composer require vendor/package-name

**Actual Result**
[RuntimeException]                                                   
  The `url` supplied for the path (repos/package-name) repository does not exist

**Comment by artmouse** [comment]
Try this:
1. bin/composer config repositories.REPO_NAME composer REPO_URL
2. bin/composer require vendor/package-name

**Comment by Ylmzef** [comment]
@artmouse it's local repo, It is located under the src folder.

**Comment by axxox** [comment]
> @artmouse it's local repo, It is located under the src folder.

For myself, I've extended `src/composer.json` by adding under `repositories`:
```json
{
"repositories": [
        {
            "type": "path",
            "url": "packages/*/*",
            "symlink": true
        }
    ]
}
```

Then the additional packages are stored in: `src/packages/namespace/package-name/` with their `composer.json` files stored there.

Then (probably is skippable) I ran `composer update --lock`.

After that - `composer req namespace/package-name`.

--

Also, when I'm changing something in a package, the `bin/copytocontainer packages` command needs to be executed to affect the changes.
